src/gateways/createArchiveLoader.ts

```diff
@@ -114,33 +117,45 @@ export default (options: Options = {}) => {
   const contentUrl = (host: string, archivePath: string, ref: string) =>
     `${host}${archivePath}/contents/${ref}.json`;
 
+  // Allow loading absolute and relative paths as those could be passed to the content-styles param
+  const makeResourceUrl = (bookRef: string) => (
+    host: string, archivePath: string, resourceRef: string
+  ) => isAbsoluteUrl(resourceRef) ?
+         resourceRef :
+         /^\.{0,2}\//.test(resourceRef) ? // Absolute path or relative path starting with ./ or ../
```

Comment:
I'm not really liking the handling of the non resource version of the stylesheet here, but I see what you set it up this way, let me think about it for a little bit

Reply:
@Dantemss can you move only the query handling part of it back to the react component? I really like the way all this is set up for the book data and book stylesheet, but the query is a special case and we don't care about it being prerendered or anything

Sorry about the rework on this

Reply:
So you want the loader to only be called when not using the param, and I'm guessing just assume all urls passed to the loader are relative to the book url?

Reply:
Yes

And its ok for the query param to not be integrated into prerender and/or cause style flash, that is just a development tool

Reply:
Alright, I restored the query param logic and the useEffect for that case.